fairseq/data/dictionary.py

```diff
@@ -125,6 +125,7 @@ def finalize(self, threshold=1, nwords=-1, padding_factor=8):
             i = 0
             while threshold_nwords % padding_factor != 0:
                 new_symbols.append('madeupword{:04d}'.format(i))
+                new_count.append(1)
```

Comment:
Let's make this 0 instead of 1, to make it clearer that these words don't actually appear in the dataset.